webapp/package.json

```diff
@@ -30,6 +30,7 @@
     "@babel/preset-typescript": "7.16.0",
     "@formatjs/cli": "5.0.7",
     "@mattermost/client": "file:mattermost-webapp/webapp/platform/client",
+    "@mattermost/desktop-api": "https://gitpkg.now.sh/mattermost/desktop/api-types?MM-55503",
```

Comment:
Need to update this once we merge the desktop side PR.

Reply:
Yep 👍